code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -345,6 +370,7 @@
 	description = "Heals toxin damage as well as slowly removing any other chemicals the patient has in their bloodstream."
 	reagent_state = LIQUID
 	color = "#000000"
+	overdose_threshold = 0 // It fucking purges all chems in your system
```

Comment:
:-1:

as stated above if I just drink some charcoal i can now ignore 90% of ODs